db/db_impl.cc

```diff
@@ -1231,6 +1231,7 @@ void DBImpl::BackgroundCallPurge() {
       auto number = purge_file->number;
       auto job_id = purge_file->job_id;
       purge_queue_.pop_front();
+      purge_queue_filenum_.erase(number);
```

Comment:
Do you plan to replace `purge_queue_` with `purge_queue_filenum_`?

Reply:
Done.